bsip-0032.md

```diff
@@ -0,0 +1,92 @@
+    BSIP: 0032
+    Title: Always Match Orders At Maker Price
+    Author: Abit More <https://github.com/abitmore>
+    Status: Draft
+    Type: Protocol
+    Created: 2018-02-16
+    Discussion: https://github.com/bitshares/bitshares-core/issues/338
+    Replaces: -
+    Worker: To be done
+
+# Abstract
```

Comment:
IMO most of this Abstract belongs in the Motivation section.
Abstract is only a concise overview of the BSIP.
Motivation explains Why the change is proposed.
Rationale explains specific decisions that were made within the Specification.